Retire G4/G5

Navid200 opened the pull request:
Can you tell me what you think of this?

I can change anything you don't like.

| Before | After |  
| ------- | ------ |  
| ![Screenshot_20250130-205023](https://github.com/user-attachments/assets/bd69eb77-a760-43ff-8f62-042ded862a66) | ![Screenshot_20250313-224626](https://github.com/user-attachments/assets/47d0e133-9bfd-40f0-9ccc-f843e30fb760) |  
| ![Screenshot_20250130-205044](https://github.com/user-attachments/assets/6cd417da-622f-48ae-9c8a-69acc00892ba) | ![Screenshot_20250312-115136](https://github.com/user-attachments/assets/0fa076ee-6354-463d-9223-aa610ded4b63) |  
| ![Screenshot_20250313-230053](https://github.com/user-attachments/assets/70b055ca-0242-455f-82a2-c85aab11982e) | ![Screenshot_20250313-224716](https://github.com/user-attachments/assets/af59d285-874d-471a-b56c-f1099516412b) |  
| ![Screenshot_20250130-205110](https://github.com/user-attachments/assets/7677053b-0ba0-47f8-beeb-46327636b88b) | ![Screenshot_20250130-204701](https://github.com/user-attachments/assets/cb999168-e0fc-41ec-a334-2e29a2ab9fae) |  


I have tested this and it is ready for review.

Navid200 commented:
There are people who are still using G5.  This I suspect is partly because they see no reason to use them and keep them just in case.  

If we retire G5, it doesn't mean no one can use G5 any longer.  The older releases can still be used.  It is not a good idea to force people to use our older versions.  But, our current stable release supports G5.  I am not suggesting that they use a 2 year old release.  As long as we continue to support G5, people will have no reason to use their sensors.  They may never use them.  
Their thinking is if some days something happens that causes them not to get G7, they have something to get them going.  if something like that happens, they can use an older version of xDrip as it is an emergency case.  it is not something that is going to continue for weeks.  it is something until they resolve the hold up on their new supplies.

Navid200 commented:
This comes up often when someone cannot find the specific device name in source wizard.
<br/>  
![Screenshot 2025-03-01 080933](https://github.com/user-attachments/assets/88421020-83b6-4d0e-976d-8a735c602579)
<br/>  
  
This PR addresses that by having everything under Dexcom.

jamorham commented:
Yes we can do this but I don't want to use trademarked names if we can avoid it, so I would prefer Dex and picture of G6 and G7 as options.

Navid200 commented:
Is it OK now?

Navid200 commented:
It is not clear since I am showing two devices in the image.
I can add two different options one for G6 and one for G7 both choosing the same setting.  But, that is going to cause the misunderstanding that if someone chooses one, they will have to change a setting to switch to the other, which will be incorrect.

Can the image just show a frame with Dex shown in the middle?  So, the title will be Dex and the corresponding image will also show Dex in a frame.

Navid200 commented:
I added a border around each one of the two images hoping it would make it clear that the option Dex stands for either G6 or G7.

Under Hardware Data Source, I have changed G6/G7/1/1+ Transmitter to Dex.

jamorham commented:
Okay this looks good but I need confirmation that you own the copyright for the image such that you can submit it in to the project where it will be under the GPL license. Typically this means that the original source of the image is your own work/photograph and hasn't been taken from another source?

Navid200 commented:
I took the two photographs.  I then used Paint in Windows to merge them together into a single image.
It is my work and I am happy for it to be used in xDrip.  That's why I created it.
I have not used any material from the web or anywhere else.

jamorham commented:
Ok thanks that's what I needed.
